rclpy/src/rclpy/event.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <pybind11/pybind11.h>
+
+#include <rcl/error_handling.h>
+#include <rcl/rcl.h>
+#include <rcl/time.h>
+#include <rcl/types.h>
+
+#include <condition_variable>
+#include <cstring>
+#include <memory>
+#include <stdexcept>
+
+#include "event.hpp"
```

Comment:
there's already a bunch of different things called event here and there.

Examples:

- https://github.com/ros2/rcl/blob/master/rcl/include/rcl/event.h
- https://github.com/ros2/rclcpp/blob/master/rclcpp/include/rclcpp/event.hpp

Could we use a more specific name?
ClockEvent?

Reply:
`Event` -> `ClockEvent` in cd90a7765306da3021c41629b21b640d74e2436f and dbf83fe13005b8cdb1e3dd83ecacb6706501b7e2